libs/sdk-ui-charts/src/highcharts/chartTypes/waterfallChart/tests/waterfallChartOptions.test.ts

```diff
@@ -123,41 +119,10 @@ describe("waterfallChartOptions", () => {
 
             expect(seriesData[seriesData.length - 1].isSum).not.toBeDefined();
         });
-
-        it("should add the shadow colum for the metric is total", () => {
```

Comment:
Should we rather skip the removed tests and migrate later?

Reply:
waterfallChart was not possible create on goodsales same way some cases not nested now